.gitignore

```diff
@@ -1,3 +1,11 @@
+#Mac OS 
 .DS_Store
 *.DS_Store
-**/.DS_Store
\ No newline at end of file
+**/.DS_Store
+
+# Google SPS
+target/
+.idea
+
+# dependencies
```

Comment:
Can you also add `Webpage/node_modules`? It's really hard to review this when it says 33,634 files have changed 😛